environment.yml

```diff
@@ -4,6 +4,7 @@ channels:
   - cta-observatory
   - conda-forge
 dependencies:
+  - python=3.7
```

Comment:
why would we be testing against python 3.6 and requiring 3.7?

Reply:
Again, what is in this environment is not what you require. It is an environment specification.
You can only have a single version of python in an environment.

Which python version you require is set in setup.py (`python_requires` I think), and ctapipe currently supports `>=3.6`, so I thought here it is the same.